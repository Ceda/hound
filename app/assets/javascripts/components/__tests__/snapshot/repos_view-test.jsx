import ReposView from '../../repos_view.js';

it('renders appropriately', () => {
  const organizations = [
    { id: 1, name: "Test org" }
  ]
  const repos = [
    {
      id: 1,
      name: "Test repo",
      owner: {
        id: 1
      }
    }
  ]

  const component = renderer.create(
    <ReposView
      isSyncing={false}
      organizations={organizations}
      repos={repos}
      filterTerm={""}
      onRepoClicked={(event) => onRepoClicked}
      isProcessingId={null}
     />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
