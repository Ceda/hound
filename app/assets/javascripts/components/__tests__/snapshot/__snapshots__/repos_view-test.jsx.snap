exports[`test renders appropriately 1`] = `
<ul
  className="organizations">
  <div
    className="organization"
    data-org-name="Test org">
    <header
      className="organization-header">
      <h2
        className="organization-header-title">
        Test org
      </h2>
    </header>
    <section
      className="repo_listing">
      <ul
        className="repos">
        <li
          className="repo">
          <div
            className="repo-name">
            Test repo
          </div>
          <div
            className="repo-activation-toggle">
            <div
              className="repo-restricted">
              Only repo admins can activate
            </div>
          </div>
        </li>
      </ul>
    </section>
  </div>
</ul>
`;
